src/scss/mixins/components/_button.scss

```diff
@@ -38,15 +38,18 @@
         #{if($compat, '&-rounded-sm', '&.-rounded-sm')},
         #{if($compat, '&-rounded-lg', '&.-rounded-lg')} {
             height: const(btn-height) - 12;
+
+            .ns-android & {
+                margin: const(btn-margin-y) const(btn-margin-x);
+            }
+        }
+
+        #{if($compat, '&-rounded-sm', '&.-rounded-sm')} {
```

Comment:
On close inspection, the small border-radius is the same as the Android default border radius of 2pt. I would rather make it the same across both OSes and remove the * 2 override completely. However, since the default small border-radius doesn't do anything on Android (compared with the default) - we might be best to up the small radius to 4 in variables/_index. Consulted with our designer and she agreed to that.

So to summarize - up $border-radius-sm to 4, remove the * 2 override completely to fix this issue. :)